### ISSUE TITLE
MPI sub-jobs in Swift/T+Flux

### ISSUE THREAD

**Issue by jmjwozniak** [ExaWorks/SDK#19]
An example of how we create external MPI sub-jobs from Swift/T is here:

https://bitbucket.org/mdorier/mpix_launch/src/master/example

The idea is that the child MPI app can be run on any subcommunicator from the parent MPI app.

If we could get a similar example working with Flux, it will be simple to bring that into Swift/T.

This all depends on #12 and #13 .

**Comment by dongahn** [comment]
Yes, this is excellent. I do have an Univ. of Utah collaboration who wants to try Flux+Swift/T to speed up a floating point analysis search/testing workflow (FLiT Bisect tool: see https://cacm.acm.org/magazines/2021/2/250083-keeping-science-on-keel-when-software-moves/fulltext?mobile=false). 

Once we have some level of integration, I can ask the student to try this.
